src/component/Post.jsx

```diff
@@ -0,0 +1,37 @@
+const Post = () => {
+  return (
+    <>
+      <section className="bg-white  flex p-4">
+        <img
+          src="./assets/social.jpg"
+          className="rounded-full w-10 h-10 mr-2 "
+          alt=""
+        />
+
+        <div className="flex flex-col w-full">
+          <input
+            placeholder="start your post..."
+            className=" h-20 bg-gray-100 px-2 "
+          ></input>
```

Comment:
Input is a self-closing tag. no need to use like this

Reply:
yes I missed that to do.